Cypress/cypress/pages/workflow.js

```diff
@@ -107,3 +108,40 @@ Cypress.Commands.add("verifyDetails", (name, description, schedule) => {
     cy.get("[data-cy=schedule]").should("have.text", "Scheduling now"); // Schedule Validation
   // cy.get("[data-cy=AgentName]").should("have.text", "Self-Agent");
 });
+
+/// ************************** Validate verdict of given workflow and agent **********************
+
+Cypress.Commands.add("validateVerdict", (workflowName, agentName, expectedVerdict) => {
```

Comment:
@DhananjayPurohit Can we also take RR score & no. of expected passed experiments & verify the same in this function.

Reply:
@Jonsy13 I have added the above things into the function.

Reply:
Yes, function looks good now.